sns_ik_lib/matlab/snsIk_acc_QP_mt.m

```diff
@@ -0,0 +1,130 @@
+function [ddq, sData, exitCode] = snsIk_acc_QP_mt(ddqLow, ddqUpp, ddxGoalData, JData, dJdqData, alpha)
+% [ddq, sData, exitCode] = snsIk_vel_QP(ddqLow, ddqUpp, ddxGoalData, JData, dJdqData, alpha)
+%
+% This function implements the generic multi-task version of SNS-IK acceleration
+% solver using QP.
+%
+% INPUTS:
+%   ddqLow: lower limit for JDataoint accelerations
+%   ddqUpp: upper limit for JDataoint accelerations
+%   ddxGoalDataData: task-space accelerations of all the tasks in the order of priorities
```

Comment:
Please add more data here. What is the data type of this input? How is it constructed and deconstructed?

Reply:
I did for all the functions.